src/styles/navigation.scss

```diff
@@ -0,0 +1,55 @@
+.navigation {
+  margin-right: 50px;
+  font-family: "Roboto", sans-serif;
+
+  &__items-list {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+
+    &__item {
+      margin: 0 10px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+
+    font-weight: 500;
+    line-height: 14px;
+    color: #000;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  .is-active {
```

Comment:
The .is-active class is defined twice.

```
.is-active {
  color: #00acdc;
  position: relative;
}
```

For instance, &__items-list and &__item are fine, but using a different pattern like .is-active might be inconsistent. Consider &__item--active for consistency.

```
  &__item--active {
    color: var(--main-color);
    position: relative;
  }

  &__item--active::after {
    display: block;
    position: absolute;
    height: 4px;
    width: 100%;
    bottom: 0;
    border-radius: 8px;
    content: "";
    background-color: var(--main-color);
  }
```